tests/ui/test_add_capacity_ui.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from ocs_ci.ocs.ui.infra_ui import InfraUI
+from ocs_ci.framework.testlib import ui, ignore_leftovers, skipif_lso
+from ocs_ci.ocs.resources.pod import get_osd_pods
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.helpers.helpers import wait_for_resource_state
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_bm,
+    skipif_external_mode,
+    skipif_bmpsi,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class TestAddCapacityUI(object):
+    """
+    Test Add Capacity on via UI
+
+    """
+
+    @ui
+    @skipif_lso
+    @skipif_bm
+    @skipif_bmpsi
```

Comment:
instead of 2 above skips, you can use @cloud_platform_required (if it works only on aws, gcp ...)

Reply:
it is work on vmware (internal cluster) too